src/pages/blog/[...page].astro

```diff
@@ -19,19 +20,9 @@ const { data, lastPage, url } = page
 	</section>
 
 	<section class="blog-wrapper" v-if="posts">
-		{data.map((article: any) => {
-			const imgSrc = resolvePath(article.frontmatter.image) || resolvePath('/images/' + article.frontmatter.permalink + '.jpeg')
-			return (
-				<article class="c-article">
-					<a href={resolvePath(article.frontmatter.permalink)}>
-						<img class="c-article-image has-deep-box-shadow" src={imgSrc}/>
-						<h2 class="c-blog-title">{article.frontmatter.title}</h2>
-						<p>{article.frontmatter.description || 'TODO'}</p>
-						<div class="c-info">{article.frontmatter.author || "Greg Slepak"}<time v-if="article.frontmatter.date">{ article.frontmatter.date }</time></div>
-					</a>
-				</article>
-			)
-		})}
```

Comment:
DRYed this into `BlogPostEntry.vue` so that the same UI can be used in `/tag/string` page too.